src/isIsogram.test.js

```diff
@@ -6,4 +6,24 @@ describe('isIsogram', () => {
   it(`should be declared`, () => {
     expect(isIsogram).toBeInstanceOf(Function);
   });
+
+  it(`enters izogram`, () => {
+    expect(isIsogram('horse')).toBe(true);
+  });
+
+  it(`enters not izogram`, () => {
```

Comment:
The description `enters not izogram` is not clear. Consider renaming it to something more descriptive, like `returns false for non-isogram word`.